.github/workflows/push-oci-pr.yml

```diff
@@ -19,18 +35,24 @@ jobs:
       packages: write
     steps:
       - name: Download mapt assets
-        uses: actions/download-artifact@v4
+        uses: actions/download-artifact@cc203385981b70ca67e1cc392babf9cc229d5806 # v4.1.9
         with:
-          name: mapt
+          name: mapt-amd64
+          run-id: ${{ github.event.workflow_run.id }}
+          github-token: ${{ github.token }}
+      - name: Download mapt assets
+        uses: actions/download-artifact@cc203385981b70ca67e1cc392babf9cc229d5806 # v4.1.9
+        with:
+          name: mapt-arm64
```

Comment:
I think you can use `pattern` instead so you can download both on the same action https://github.com/actions/download-artifact?tab=readme-ov-file#usage